src/css/_legend.scss

```diff
@@ -4,6 +4,7 @@
   border-radius: 5px;
   // Make clear that moving the legend will move the map itself.
   cursor: grab;
+  box-shadow: 0px 0px 5px rgba(0, 0, 0, 0.3); /* The box shadow */
```

Comment:
I'm personally anti box shadow hah. It goes against "flat design". I intentionally didn't add it even though the current map has it.

Reply:
I mainly added the shadow because if the popup is below the legend, it looks funny without the shadow. For example, the long ones like Oregon, OR.

Perhaps I have the popup above the legend?

Reply:
Yeah, I think popup should be above the legend since you can close out of the popup but you can't close out of the legend. And when the popup is there, the legend and map itself are less relevant.